CHANGELOG.md

```diff
@@ -139,6 +139,7 @@ consistent with the behavior of `collection_jitter`.
 - [#1600](https://github.com/influxdata/telegraf/issues/1600): Fix quoting with text values in postgresql_extensible plugin.
 - [#1425](https://github.com/influxdata/telegraf/issues/1425): Fix win_perf_counter "index out of range" panic.
 - [#1634](https://github.com/influxdata/telegraf/issues/1634): Fix ntpq panic when field is missing.
+- [#1642](https://github.com/influxdata/telegraf/pull/1642): input/mysql: Allow to configure whether to gather global variables and statuses
```

Comment:
Remove this as we update the changelog when we merge in order to prevent merge conflicts.